src/Symfony/Component/Debug/FatalErrorHandler/OutOfMemoryFatalErrorHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Debug\FatalErrorHandler;
+
+use Symfony\Component\Debug\Exception\FatalErrorException;
+use Symfony\Component\Debug\Exception\OutOfMemoryException;
+
+/**
+ * ErrorHandler for out of memory errors.
+ *
+ * @author error56 <nalldev2@gmail.com>
+ */
+class OutOfMemoryFatalErrorHandler implements FatalErrorHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handleError(array $error, FatalErrorException $exception)
+    {
+        if (false !== strpos($error['message'], 'memory')) {
+            $message = 'I didn\'t get enough memory for execution :\/';
```

Comment:
no need for the $message variable + let's improve the message itself, we never use `I` there :)

Reply:
ok